src/path_call_builder.js

```diff
@@ -0,0 +1,25 @@
+import {CallBuilder} from './call_builder';
+
+/**
+* @class PathCallBuilder
+*/
+export class PathCallBuilder extends CallBuilder {
+    /*
+    * @constructor
+    */
+    constructor(url, source, destination, destination_type, destination_amount) {
+        super(url);
+        this.url.segment('paths');
+        this.url.addQuery('destination_account', destination);
+        this.url.addQuery('source_account', source);
+        this.url.addQuery('destination_amount', destination_amount);
+
+        if (!destination_type.isNative()) {
+            this.url.addQuery('destination_asset_type', 'credit_alphanum4');
```

Comment:
Should that logic go in `js-stellar-base` instead? I started that [here](https://github.com/stellar/js-stellar-base/compare/master...masonforest:get-asset-type?expand=1). I'm not sure if it should return a constant or a string. Let me know and I'll open a PR on that.